Dockerfile

```diff
@@ -0,0 +1,20 @@
+from node:lts as build
+workdir /app
+copy . .
+run mkdir certs && openssl req -x509 -newkey rsa:2048 -sha256 -days 36500 -nodes -keyout certs/privkey.pem -out certs/fullchain.pem -subj '/CN=dialog'
+run npm ci
```

Comment:
See my comments in the Hubs repo, I think we can probably improve caching for builds by separating out the `npm ci` step from the rest of the build.